frontend/src/views/Lobby.tsx

```diff
@@ -48,6 +53,15 @@ function LobbyPage() {
     console.log(user);
   };
 
+  const handleClick = (): Promise<void> => {
+    const request = { roomId: currentRoomId, User: currentUser };
+    return axios.post<void>('/api/v1/start', request)
+      .then(() => { console.log('redirect to game'); })
```

Comment:
nit: Remove `console.log(...)` and replace it with a `setLoading(...)` function that works as it does on other pages (you will need to add a React element to the actual function return as well for this).